tableau-mkt/github-release-bot
# Test commit 3

## What? Why?
See #9570 — it was an oversight that we did not update the target Eloqua form for the blog subscription form, which gets rendered on `/about/blog` 

## Developer Notes
- I reused the variable `eloqua_subscription_center_form_name` since it already has an admin UI and the fallback is the new subscription form name, `tableausubscriptioncenter`

## Stakeholder Notes
Stakeholders: Nareman

Updated form destination for blog subscriptions to match subscription center — `tableausubscriptioncenter`

## QA

https://success9570-tableau.pantheonsite.io/about/blog

- Submit an email address at the bottom and click subscribe
- You should see a success message
- Nareman and I already confirmed it gets passed through to Eloqua, so we just want to make sure the experience continues to work as expected


![Pasted_Image_7_8_20__1_47_PM](https://user-images.githubusercontent.com/153132/86968782-99678000-c121-11ea-9f77-c9e3c45f2ecd.png)

### Testing
- Manually, on multidev, with Nareman to confirm it passed through the submission to Eloqua as expected

## Documentation
- none

## Deployment Steps
- N/A

## Related
- Closes #9570 

## Review By
- ASAP please :)

---

**Security**
- [ ] Affects user input handling or display.
- [ ] Involves raw SQL queries.
- [ ] Changes how redirects work, eg: `drupal_goto()`, `$form_state['redirect']`, etc.
- [ ] Affects lead, customer, user, or other sensitive data.
- [ ] Involves secure or authenticated communication with other systems.

**Communication**
- [ ] Affects user-facing strings for localization.
- [ ] Affects editor UX and thus training, on-boarding, etc.
- [ ] Modifies Eloqua details or `dataLayer`, affecting marketing operations.
- [ ] Needs new/revised web developer documentation.

**Drupal 8 Migration**
- [ ] Modifies fields, taxonomy, etc., which affects D8 migration or existing dashboards. [Field Mapping Google Sheet](https://docs.google.com/spreadsheets/d/1uZ-xJDI4mFcDi1LWxYgk9TRWp9VcPX4azKs2xoG7YiI/edit#gid=1831588720)
- [ ] Removes or adds a view or view display [Views Mapping Table](https://docs.google.com/spreads
